src/templates/page.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import { graphql } from 'gatsby'
+import Layout from '../components/layout'
+
+/**
+ * Generic WP page template.
+ */
+class PageTemplate extends Component {
+  render() {
+    const currentPage = this.props.data.wordpressPage
+
+    return (
+      <Layout>
+        <h1 dangerouslySetInnerHTML={{ __html: currentPage.title }} />
```

Comment:
Am assuming this is `the way` to include wordpress content.

Is it possible to import as plain text rather than html? I'm assuming for our purposes we only want to keep the string content - not any surrounding markup.

Reply:
This is `a way`, specifically the way that the demo project I lifted the bulk of that code from did it. I think perhaps that this should be a separate discussion as there are many potential solutions here.

I'd hypothesize that it's done this way as it allows WP to handle all the preprocessing of content and just pass us the rendered HTML for stuff like markdown, images, emphasized text etc..

Reply:
Would be good to get other opinions here. I'd prefer to start without expecting to render html from the WP.

Reply:
I think there's a way to get markdown from wordpress instead of the raw HTML, I'll have a look at the demo I worked near the beginning of the new website.